test/library/packages/UnrolledLinkedList/insert/testInsertArray.chpl

```diff
@@ -0,0 +1,27 @@
+private use UnrolledLinkedList;
+
+config const testIters = 8;
+
+var arr: [0..#testIters] int = 1..testIters;
+
+var lst: unrolledLinkedList(int);
+
+for i in 1..testIters do
+  lst.append(i);
+
+for (x, y) in zip(arr, lst) do
+  assert(x == y);
+
+assert(lst.insert(lst.size, arr));
+
+assert(lst.size == 2 * arr.size);
+
+for i in 0..#testIters do
+  assert(arr[i] == lst[i]);
```

Comment:
Was this meant to be testing `arr[i] == lst[testIters + i]`?

Reply:
Yeah